src/atoms/Spacer/index.js

```diff
@@ -14,20 +14,16 @@ function Spacer( props ) {
     className
   } = props;
 
-  const variants = classnames(
-    { [styles.xSmall]: xSmall },
-    { [styles.small]: small },
-    { [styles.medium]: medium },
-    { [styles.large]: large },
-    { [styles.xLarge]: xLarge },
-  ).split( ' ' );
+  const classes = [
+    xSmall && styles['x-sm'],
+    small && styles['sm'],
+    medium && styles['med'],
+    large && styles['lg'],
+    xLarge && styles['x-lg'],
```

Comment:
@patrickkim If you're going to change this syntax here, I think it should also be changed in the `Layout` component for consistency.